lua/rzls/init.lua

```diff
@@ -87,20 +84,18 @@ function M.setup(config)
                     razor.apply_highlights()
                     documentstore.register_vbufs(bufnr)
                     rzlsconfig.on_attach(client, bufnr)
-                    if not client.hacked_semantic then
+                    if not client.hacked_capabilities then
```

Comment:
Should we use something like `client._rzls_hacked_capabilities` instead of just `client.hacked_capabilities`? Maybe I'm being paranoid, but I can see people using `hacked_capabilities` in their own `on_attach` handlers